src/main/java/com/spikes2212/command/drivetrains/commands/DriveArcadeWithPID.java

```diff
@@ -1,61 +1,91 @@
 package com.spikes2212.command.drivetrains.commands;
 
 import com.spikes2212.command.drivetrains.TankDrivetrain;
-import com.spikes2212.control.PIDLoop;
+import com.spikes2212.control.PIDSettings;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.controller.PIDController;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
 import java.util.function.Supplier;
 
-
+/**
+ * A command that moves a drivetrain with a set speed forward and with a PID loop to a certain angle.
+ */
 public class DriveArcadeWithPID extends CommandBase {
```

Comment:
please add an option for a `FeedForwardController` to this command. This will be done by adding an optional constructor that receives an instance of `FeedForwardSettings`, and one that doesn't.

Reply:
For forward movement or for turning?